composer.json

```diff
@@ -13,13 +13,14 @@
             "name": "Nikita Prokurat",
             "email": "nickpro@tut.by"
         }
-    ],     
+    ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.0"
+        "php": "^5.6",
+        "symfony/framework-bundle": "~3.4",
```

Comment:
I would recommend same approach with versioning as proposed here
https://github.com/clarity-project/ClarityImagesBundle/pull/2#discussion_r530886214

Unless we have a reason to drop support for 2.x symfony. 
I personally use this bundle with 2.x is some old projects.